charts/nexus-iq/values.yaml

```diff
@@ -18,7 +18,7 @@ iq:
p+KUXzNi4iiMjKB4xdpyZAIpJbTe+1PtUUcmZL4cW0JigmEF3J5WKoTY+dX2dkxBhxgHfVoq6V77XiiSR+idQ671Fci/GTq6efjDQDyokYc/Lv7zvHRtj5lX/JFjaatSP7jjIEdiyEmIeBB0Enqo2G2LpTSuhXSYJKMagygME5dAQxohmoVMaklLT/VIKwaUNOHi179cB0UpHzxRCJLdhzsMW16QamJZQ3hUeNIwhTt83eBjJOsm+bgPZ9xbKQZ20IsXyCF+qAnRs3h4W8xHkmE9pfBwb/z1fN/fjmXCW95eh3Z6hiG/D0RjcyzzAz1KtZn/CfGvSnEgQnk9njrZ77aRgbcwZDTP4m656oZvNbr/WriCRQUnbykkmsdz7RvyJ8tMPZtPHd/b7XsMRNQd54rJ9YyWJcIhHgXT3azQ7zNbcN05NONtpeuDb73s6nGlxEp8HH7GdauNRU3dcFxVoQTAUswVYV6mxzn3XWX1WIZBG7MUv2P/hOX/YWy/jhxmgtQM4pfoji2jGckKbQ3b8zpB2wX1FE27xuEul+kelkCOSCGgchdNNPBFSopvqpyID/G4ssJPXYyFro+sG5YHLyqmSmooYvtg0HkfS/n+eB8r43HcNF5QPevG1vReVdauirPiWTKd5U6Up2cofCUI2ZHX8X/I3pgJIiPwvZmrlMXph9Ac1LQfactGKEi+A7sPByr3ZHImujnThUzPPXN4vtZ/sa8CMs4hL0MbDo2pBNkc96yki/cfLQyb+y+27vHfh9+a3bdl6QX2JKJqn3OLocU9+YDLmXdYQtiJnZIouRl3RDxTAYaP1vt8alrEq/GepbDjkK/93g/m9fG+KSXsnNoLf3SVWnGSkLY3k5W4ajhK/SyDUIXA3SBMmZjsLVO3BE6G49Pt/75/jcIC/gdbS7IOYuYtX1EYO4VYQtnukYVQ/SKPrY60gVSAEby66ACbvdNm78nVOJ9It84YG+1NSpHNHEQMqnz2dNdNHmwweIvoS9FbhKYtxUPcuJ58WlXkkttt8GUKvXtVLh0xU/lPIkF0+Qt/59Fqnl5StxXlTuhr4abrF1+Y+AxhmFs7Y/BR13v3oii3NVa9u+RGhBqgBOD8wRnx2aEqRzWQtcXc4Yucu0SCnK6T0CfLIHKEX0V7JiI/Hm/4Hj7QWi3XiDAbOE8a2dtTu7U5aiEZFHDx9+U0z0iQsJLErJZk5NTXZUUILoWSfF6F9Q+G3thFc+4+wYw3Cb4Q2drKMSVUm4JVMYTR3fWNaICFdvqMeMowdWaO4D6APrC5amchFzFSrxEVdrlU2t225AeTfbTTVsUHN/FiPFjCHLtUF42MKJvtDCTMK4GXWUfLjDN4Hat2vXXciOnBRkyqma2zI8jJESAHyvPg4cAJqd3wOgeJsrFtt6r3MnKrzOAVZRfvraA2vKexMSiHyoTtQVMaF/5/dToJXr6a80YmhRUiH+s3eeIb8FN54dbUn3P/DmTarpzSpDMVQjx+U/9tooH4pKEosGE2CWp67E8FtDVDqc5Fi3iUTLIUOaP/heRDPYvcogz8jAj546JC1ZAQXVUWoKJKsSKG87dxq6ElwQvCVh4RoaNM4ilNqQxh0xILnlGu06U6xCB281LyQdJitkfvdoUbdXbqcX07bV5HoRnRarDu5bI6r7e9/x39NPqgOl6yjEpD4IK/Su2YQvGRCym6pnr9+DjDSplvmNHr/1MAuxUkTfQd5DpnyuiTxQHS3Ewn5svhY4MziEfsAy50GrwjamfiFbOhxfR2IlVZ3BaPJdbsD52ju6st2iXpuOzPwY0/v2szJu4lfj/eR2h9jimD9zEsdS6yNDwRLZH7ISGk0XTtmZjQLX+bV8U6XRw2+/P31CuNa8qOiZiwnHUkEkb4LnHQhs+6/VuD+A5IBAnk7K8ZHbdiq86ttUHJnZ3vv8KVvXQvcZF5xlY6PkPTa70HqQz+STHPhTctkTyJeXIC7273uqcDGLTiDfC2gBtpHhbe98tiJaXeOexYZf8Seg6MZ5ixyZi+zAhJ1rcGN5QHeXH6r70OA5YfquroHsmxp1f4eNCJwpyHY25hcJaJuSGoPdFJra1fMc9DjosIzZNxcbn4236gB264qxk/T+pZXbzgdZdA40PpcaP0Gv2CwyihgiubRd/+JDBM62o3me9cAAIlQm1tbapY/fTN4gbogkfzOjYMHTCqbIN2koY0NESFWOmE+wk22IQiT/ME3iX1tPlPOAIz+0Rt9JL1mA8fw3vb7XOSAoVvOytuuuTLg3+TKjirda7cMcupkTrQN14B7i/zqmO6Dxeu8oj0ShcSuWC+2VCj8BM6/lAPN7BdqMSSsfAEp78AFa5mN1fnoj9vcKrzps2zUKUoQSBPKtFRUgG7Vh821HmIKcm0OLHvcJAwo7uNrECXUdRsAEY5CC+H5XnQnS+rMUsu5+3KjEUu2R2HdSpyMrh1oOMz9UN6hbf/lMMCTYqinfBYn/Iv8Iw8MP5KbN4KsGroxzpIwxHSANbJSAKB8jvLk/Xdc3OLXxlZSpK0ZdrmUOt+MH61wlmCtsOkhM4Va/HzR+DOHckznOSee6r7niYFfj0JgUkMZeDYRQ5cx5tqogR8HHn2drp1gEDPRJewqtpEGnZJaXIB0QOgdvA52+Cmx90Xb6rHXCqZ2d0bz88blbpkbY65dhxllbENfcW8c9dKZo3vIYCQ704VswQl8buXkLZPjR0aiLLZTtWf3+NSVeqrwsM9e5YHn+CnhlV909VoJFWFAnT1I3wnKM2u6O2ZRNmaAV6R4+W++DAopqFrF7+hPxo0K6jZxbUCodVn0wZuvhEiXf6n378V43pvDqudOrbJo9Za7ScMLSkTy3JePGmnGqKsBZ0Op7VBQ2FOTyaszpe5xZm2bwq39tjn/qkn3orbW6qhNvpdgdoxddUix/aqKjO5b/5k2WLnrqbS6mJx/kUM3BUdCSm8qB6ankQyzMplD2U5tIcqer9xFmIvWCK57mHSytu3A=="
```

Comment:
Is this an actual license? That doesn't seem right. Actually, since this is a public repo, if this _is_ a legit license, we may need to contact IQ & security to see if we can burn this license.

Reply:
I'll hop on the branch and see what it does. 🚨

Reply:
license managers are taking care of the exposed license, and I force-pushed an update to get it out of the clear in the meantime.